docs/development/plugin/development-plugin-localization.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[development-plugin-localization]]
+=== Localization for Plugins
```

Comment:
Several paragraphs in this doc include lists.  For the lists to work, add a line space above the list.

Also, some of the links didn't work.